src/stylesheets/elements/_buttons.scss

```diff
@@ -80,6 +80,37 @@
   background-color: transparent;
   box-shadow: none;
 
+  // Add color variants here
+  &.usa-button--secondary {
+  	@include u-border("secondary");
+    background-color: white;
```

Comment:
The mockup shows semi-bold text in the "ink" color, so we need to add:
@include u-text('semi-bold', 'ink');